Sources/GeneralIOs/Commands/Font.swift

```diff
@@ -0,0 +1,287 @@
+//
+//  File.swift
+//  GeneralIOs
+//
+//  Created by Evgeny Schwarzkopf on 29.03.2022.
+//
+
+import Foundation
+import ArgumentParser
+import GeneralKit
+import AEXML
+import Stencil
+import PathKit
+
+public final class Font: ParsableCommand {
+
+    private enum Constant {
+        static let fontsFolderPath = "./Resources/Fonts"
+        static let extensionFolderPath = "./Classes/Presentation/Extensions"
+        static var extensionFontPath = "./Classes/Presentation/Extensions/UIFonts+App.swift"
+        static let fontsTemplatePath = "./.templates/rsb_fonts"
+        static let commonTemplatePath = "./.templates/common"
+        static let commandTemplatePath = "./.templates"
+        static let fontValue = "UIAppFonts"
+        static let fontTemplateName = "UIFonts+App.stencil"
+        static let notFoundTemplate = #"""
+                                      Is not found folder templates in directory.
```

Comment:
I don't like that this values hardcoded in plugin. What about place it in general.yml?